pos_order_to_sale_order/models/sale_order.py

```diff
@@ -43,7 +43,7 @@ def create_order_from_pos(self, order_data, action):
         if action in ["delivered", "invoiced"]:
             # Mark all moves are delivered
             for move in sale_order.mapped("picking_ids.move_ids_without_package"):
-                move.quantity_done = move.product_uom_qty
+                move._set_quantity_done(move.product_uom_qty)
```

Comment:
Did you double-check the functionality for this change?

As I checked, the picking doesn't move to `Done` state when we choose `delivered` or `invoiced` actions.

keep this one will be better: `move.quantity = move.product_uom_qty`